interfaces/apparmor/template.go

```diff
@@ -92,6 +92,17 @@ var templateCommon = `
   # The base abstraction doesn't yet have this
   /etc/sysconfig/clock r,
   owner @{PROC}/@{pid}/maps k,
+
+  # /proc/XXXX/map_files contains the same info than /proc/XXXX/maps, but
+  # in a format that is simpler to manage, because it doesn't require to
+  # parse the text data inside a file, but just reading the contents of
+  # a directory.
+  # Reading /proc/XXXX/maps is already allowed in the base template. Also,
```

Comment:
Reading /proc/XXXX/maps is already allowed in the base template via \<abstractions/base\>.

Reply:
Fixed.